src/porepy/grids/grid_bucket.py

```diff
@@ -59,7 +59,7 @@ def __contains__(self, key: Any) -> bool:
             key (object): Object to be tested.
 
         Return:
-            True if etiher key is a pp.Grid, and key is among the nodes of the graph
+            True if either key is a pp.Grid, and key is among the nodes of the graph
                 representation of this md-grid, *or* key is a 2-tuple, with both items
                 in self._nodes.
```

Comment:
@keileg should all 2-tuples of grids be included, or only those which are exactly/at most one dimension apart?